traffic_server/plugins/astats_over_http/astats_over_http.c

```diff
@@ -343,25 +433,54 @@ static void json_out_stats(stats_state *my_state) {
 
 	if (my_state->recordTypes & SYSTEM_RECORD_TYPE) {
 		APPEND(",\n \"system\": {\n");
-		appendSystemState(my_state);
-		APPEND_STAT("configReloadRequests", "%d", configReloadRequests);
-		APPEND_STAT("lastReloadRequest", "%" PRIu64, lastReloadRequest);
-		APPEND_STAT("configReloads", "%d", configReloads);
-		APPEND_STAT("lastReload", "%" PRIu64, lastReload);
-		APPEND_STAT("astatsLoad", "%" PRIu64, astatsLoad);
+		appendSystemStateJson(my_state);
+		APPEND_STAT_JSON("configReloadRequests", "%d", configReloadRequests);
+		APPEND_STAT_JSON("lastReloadRequest", "%" PRIu64, lastReloadRequest);
+		APPEND_STAT_JSON("configReloads", "%d", configReloads);
+		APPEND_STAT_JSON("lastReload", "%" PRIu64, lastReload);
+		APPEND_STAT_JSON("astatsLoad", "%" PRIu64, astatsLoad);
 		APPEND("\"something\": \"here\"");
 		APPEND("\n  }");
 	}
 
 	APPEND("\n}\n");
 }
 
+static void csv_out_stats(stats_state *my_state) {
+	const char *version;
+	TSDebug(PLUGIN_TAG, "recordTypes: '0x%x'", my_state->recordTypes);
```

Comment:
This can be `'%#x'`

Reply:
It can, but leaving it as 0x%x preserves the existing syntax used in the rest of the file

Reply:
Fair enough